services/user/Identity/service/src/tests/tests.rs

```diff
@@ -0,0 +1,221 @@
+// Q: How is SERVICE possibly properly defined here? and how is the Wrapper the right wrapper?
+use serde_json::json;
+
+use psibase::AccountNumber;
+
+use crate::tests::test_helpers::*;
+
+#[psibase::test_case(services("identity"))]
+pub fn test_reject_invalid_scores(chain: psibase::Chain) -> Result<(), psibase::Error> {
+    match test_attest(
```

Comment:
Tests ideally should confirm that it fails as expected:
```rs
    let expected_err = "bad confidence score";
    match test_attest(
        &chain,
        AccountNumber::from("alice"),
        String::from("bob"),
        101,
    ) {
        Err(e) => assert_eq!(
            e.to_string(),
            expected_err,
            "Transaction failed with {}, but was expected to fail with: \"{}\"",
            e.to_string(),
            expected_err
        ),
        _ => panic!(
            "Transaction succeeded, but was expected to fail with: \"{}\"",
            expected_err
        ),
    }
```

This pattern is verbose and we probably need to extract it into a testing helper library, similar to what we have in C++, where the completely equivalent code would be a single line, like:
```c++
CHECK(attester.attest(subject.clone(), 101).failed("bad confidence score"));
```